src/extractLoader.js

```diff
@@ -91,7 +91,8 @@ function evalDependencyGraph({loaderContext, src, filename, publicPath = ""}) {
             exports,
             __webpack_public_path__: publicPath, // eslint-disable-line camelcase
             require: givenRelativePath => {
-                const indexOfQuery = givenRelativePath.includes("?") ? givenRelativePath.indexOf("?") : givenRelativePath.length;
+                const noDoubleQuestionMarkPath = givenRelativePath.replace(/\?\?/g, "##");
+                const indexOfQuery = noDoubleQuestionMarkPath.includes("?") ? noDoubleQuestionMarkPath.indexOf("?") : givenRelativePath.length;
```

Comment:
A shorter way of accomplishing the same thing is:

`const indexOfQuery = givenRelativePath.search(/(\?.*)?$/);`

Reply:
I love it but code is meant to be readable and understandable. I'm going to leave the decision to the maintainer.